_ap/0/index.adoc

```diff
@@ -0,0 +1,68 @@
+---
+num: 0
+title: Template 
+status: Draft
+authors: 
+  - "" # One item for each author, as github id or "firstname lastname"
+tags:
+  - "" # e.g. kafka, connectors, registry
+---
+
+// Top style tips:
+// * Use one sentence per line
+// * No unexpanded acronyms
+// * No undefined jargon
+
+// You don't have to use the following sections, but they provide a 
+// useful structure for writing a clear document.
+
+## Intent
+// Summarize in a single sentence what the pattern tries to achieve.
+// For example:
+//
+// Define a policy for exposing metrics to customers in a consistent way.
+
+## Motivation
+
+// In a few paragraphs describe the motivating factors for this pattern
+// For example:
+// 
+// Prometheus scrape endpoints are the defacto standard for exposing metric information for collection
+// However this format and the associated conventions are not sufficient in themselves to ensure
+// that metrics are exposed in a consistent way. Problems include:
+// * Insufficient guidance on metric and label naming to provide consistency and 
+//   establishing the principal of hiding implementation details.
+// * Treatment of this as a first-class API, with established mechanisms for API evolution
+// * etc.
+
+## Applicability
+
+// Call out when this pattern might apply (and when it should not apply, if relevant)
+// For example:
+//
+// This pattern applies to any service which exposes metrics as Prometheus-scrape endpoints.
+// It does not apply to services which provide access to metrics in other formats, or via
+// means other than an API (e.g. metric visualization via a system such as Grafana).
+
+## Structure
```

Comment:
I like all sections exceit this one for which the title is not speaking to me.
It feels like the core of the pattern. Propositions:

* What the pattern is
* Description
* Detail

Right now, *Description* is what speaks to me the most.